applications/ecallmgr/src/ecallmgr_registrar.erl

```diff
@@ -381,6 +382,12 @@ init([]) ->
 handle_call('registrar_age', _, #state{started=Started}=State) ->
     wh_util:put_callid(?LOG_SYSTEM_ID),
     {'reply', wh_util:current_tstamp() - Started, State};
+handle_call({'import_registrations', Filename}, _, State) ->
+    wh_util:put_callid(?LOG_SYSTEM_ID),
+    {'reply', import_registrations_from_file(Filename), State};
```

Comment:
The registrar will be blocked by the import/export action and will be unable to respond during the completion of this job.

Please spawn a separate process to actually do the work. For example here the answer can be `{reply, Ref, State}`, this way the spawned worker process can send back `{Ref, Imported}` and this reply should be caught by the caller using something akin to `catch_my_request(Ref) -> receive {Ref, Result} -> Result end.`